zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -166,12 +163,20 @@ impl<F: FieldExt> ExecutionGadget<F> for BeginTxGadget<F> {
             ),
             (CallContextFieldTag::Value, tx_value.expr()),
             (CallContextFieldTag::IsStatic, 0.expr()),
+            (CallContextFieldTag::LastCalleeId, 0.expr()),
```

Comment:
want to know why call last callee in begin_tx ?

Reply:
To avoid a malicious prover set these fields to read arbitrary call's memory, we need to make sure it's initilized to `0` properly